### ISSUE TITLE
Scalability tests broken

### ISSUE THREAD

**Issue by jkuester** [medic/cht-core#8962]
**Describe the issue**
[Noticed this](https://medic.slack.com/archives/C024KTGRW/p1711040146261659) during the `4.6.0` release process.  The scalability test suite appears to be broken and all the tests are failing (even though the CI job succeeds).  

The tests all fail with an error like:
```
(node:9140) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
(Use `node --trace-warnings ...` to show where the warning was created)
initial replication failed Error
    at generateErrorFromResponse (/cht/cht-core/tests/scalability/node_modules/pouchdb/lib/index.js:694:18)
    at /cht/cht-core/tests/scalability/node_modules/pouchdb/lib/index.js:6816:19
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  error: 'forbidden',
  reason: 'You are not allowed to access this db.',
  status: 403,
  name: 'forbidden',
  message: 'You are not allowed to access this db.',
  docId: '_local/c8MWxDNnf3o5z.zmupV9KQ%3D%3D',
  result: {
    ok: false,
    start_time: '2024-03-21T15:42:33.844Z',
    docs_read: 0,
    docs_written: 0,
    doc_write_failures: 0,
    errors: [],
    status: 'aborting',
    end_time: '2024-03-21T15:42:33.885Z',
    last_seq: 0
  }
}
```

The JSON results from the run look like:

```
{
  "Total" : {
    "transaction" : "Total",
    "sampleCount" : 10,
    "errorCount" : 10,
    "errorPct" : 100.0,
    "meanResTime" : 10923.900000000001,
    "medianResTime" : 11603.0,
    "minResTime" : 6424.0,
    "maxResTime" : 12100.0,
    "pct1ResTime" : 12089.6,
    "pct2ResTime" : 12100.0,
    "pct3ResTime" : 12100.0,
    "throughput" : 0.7927699381639448,
    "receivedKBytesPerSec" : 0.0,
    "sentKBytesPerSec" : 0.0
  },
  "sync" : {
    "transaction" : "sync",
    "sampleCount" : 10,
    "errorCount" : 10,
    "errorPct" : 100.0,
    "meanResTime" : 10923.900000000001,
    "medianResTime" : 11603.0,
    "minResTime" : 6424.0,
    "maxResTime" : 12100.0,
    "pct1ResTime" : 12089.6,
    "pct2ResTime" : 12100.0,
    "pct3ResTime" : 12100.0,
    "throughput" : 0.7927699381639448,
    "receivedKBytesPerSec" : 0.0,
    "sentKBytesPerSec" : 0.0
  }
}
```

Note the `"errorPct" : 100.0`.

**Describe the improvement you'd like**
We should fix the tests!  Currently we have not collected any test results since `4.3.0`!

**Comment by dianabarsan** [comment]
This is happening because the users that we use in the scalability suite don't have correctly assigned roles. With the upgrade to CouchDb 3, database security is increased and CouchDb only allows writes from users/roles that have appropriate permissions. Because the scalability users lack any configured roles, they fail on writes.

**Comment by dianabarsan** [comment]
As can be seen here, users have the `district_admin` role: https://github.com/medic/cht-core/blob/master/tests/scalability/csv/users.csv
This role is not defined in config, therefore users with only this role will not be allowed to directly read or write from the database,

**Comment by dianabarsan** [comment]
For reference, I ran the scalability suite against this branch and these are the results: https://s3.console.aws.amazon.com/s3/object/medic-e2e?region=us-east-1&bucketType=general&prefix=scalability/8962-fix-scalability-tests-8430121913/statistics.json
